WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -84,6 +84,21 @@
   </div>
 </div>
 
+<!-- Hidden data range feature enabled by setting one of those parameters in the URL. -->
+<% if %i(date_range from_date to_date).any? { |key| params.key? key } %>
```

Comment:
I guess this comment should be one diff above ^^
I didn't use datepicker at this point. Do we have a nice way of enabling it without simple_form?